source/collections/capped-collection.txt

```diff
@@ -0,0 +1,55 @@
+.. _capped-collection:
+
+==========================
+Create a Capped Collection
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Capped Collections </core/capped-collections/>` are fixed-sized 
+collections that support high-throughput operations that insert and retrieve 
+documents based on insertion order. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Click the :guilabel:`Create Collection` button.
+
+      From the :guilabel:`Collections` screen, click the 
+      :guilabel:`Create Collection` button.
+
+   .. step:: Enter the collection name.
+
+   .. step:: Click the :guilabel:`Advanced Collection Options` dropdown.
+
+      Check the :guilabel:`Capped Collection` option.
+
+   .. step:: Enter the :guilabel:`size` of the capped collection.
+
+      Enter the maximum number of bytes that the collection can hold.
+
+   .. step:: Click :guilabel:`Create Collection` to create the collection.
+
+Restrictions and Limitations
+----------------------------
+
+The following restrictions and limitations apply when creating a 
+capped collection:
+
+- :guilabel:`Custom collation` is the only :guilabel:`Advanced Collection Option` 
+  that can be used alongside your capped collection.  
+
+- If you import a data set larger than the maximum size of the capped collection,
+  |compass-short| will only load the part of the data set that is within the 
+  size range. 
```

Comment:
This might sound as if Compass stops importing data once it hits the size limit; in reality, Compass keeps going, and the oldest documents are dropped, i.e. only the *last* part of the import data is included.

Reply:
Done.